app/channel.go

```diff
@@ -3007,6 +3009,19 @@ func (a *App) ClearChannelMembersCache(channelID string) {
 
 		page++
 	}
+
+	return nil
+}
+
+func (a *App) ClearChannelMembersCache(channelID string) {
+	clearSessionCache := func(channelMember model.ChannelMember) error {
+		a.ClearSessionCacheForUser(channelMember.UserId)
+		message := model.NewWebSocketEvent(model.WEBSOCKET_EVENT_CHANNEL_MEMBER_UPDATED, "", "", channelMember.UserId, nil)
+		message.Add("channelMember", channelMember.ToJson())
+		a.Publish(message)
+		return nil
+	}
+	a.forEachChannelMember(channelID, clearSessionCache)
```

Comment:
Should we log the returned error (if any)?

Reply:
Nothing in the `clearSessionCache` function returns an error, so it'll never return one in this case.

Reply:
I was thinking about `GetMembers()` failing. Before the refactor we were logging that.

Reply:
Good point. Added handling for it.